zappa/core.py

```diff
@@ -1247,6 +1247,10 @@ def create_lambda_function(
                 ReservedConcurrentExecutions=concurrency,
             )
 
+        # Wait for lambda to become active, otherwise many operations will fail
+        # https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/lambda.html#waiters
```

Comment:
Should we move this reference into the `wait_until_lambda_function_is_ready` docstring?